get_releasenote.py

```diff
@@ -27,7 +27,7 @@ def parse(
     msg = msg.strip()
     head_re = re.compile(
         head_line.format(
-            version="(?P<version>[0-9][0-9.abcr]+)",
+            version="(?P<version>[0-9][0-9.abcr]+(.post[0-9]+)?)",
```

Comment:
Shouldn't `.` be escaped as `\.`?

Reply:
Also, a full regex is a bit more complicated: https://www.python.org/dev/peps/pep-0440/#appendix-b-parsing-version-strings-with-regular-expressions.